tests/userLogin.test.ts

```diff
@@ -0,0 +1,36 @@
+import request from "supertest";
+
+import { Role } from "../src/models/user.model";
+
+import { STATUS_CODES } from "../src/utils/constants";
+
+import { PrismaUser, prismaMock } from "../singleton";
+import app from "../src/server";
+
+import ResolvedValue = jest.ResolvedValue;
+
+describe("User login", () => {
+  const email = "test1@test.com";
+  const password = "Test123Pass!";
+
+  it("Should log in an existing user", async () => {
+    const email = "test1@test.com";
+    const password = "Test123Pass!";
```

Comment:
You don't need these since you've already declared them bellow describe :)

Reply:
Oh, I see it now. Fixed.